__mocks__/zustand.ts

```diff
@@ -0,0 +1,49 @@
+import { act } from '@testing-library/react'
+import * as zustand from 'zustand'
+
+const { create: actualCreate, createStore: actualCreateStore } =
+  jest.requireActual<typeof zustand>('zustand')
+
+// a variable to hold reset functions for all stores declared in the app
+export const storeResetFns = new Set<() => void>()
```

Comment:
je suis étonné / déçu que la lib ne fournisse pas de mécanisme out of the box

Reply:
Ce mock est celui fourni par la lib : https://docs.pmnd.rs/zustand/guides/testing#jest

Reply:
c'est copié / collé de la doc de la lib, si la lib a besoin de faire évoluer le mock, ça ne sera pas automatique

c'est décevant que la lib ne le fournisse pas